tests/unit/helpers.js

```diff
@@ -267,3 +275,39 @@ export class ErrorWithCode extends Error {
     this.code = code || 'SOME_CODE';
   }
 }
+
+/*
+ * A class that implements an empty IExtensionRunner interface.
+ */
+export class FakeExtensionRunner {
+  params: any;
```

Comment:
Couldn't this also have a real constructor like:

```js
export class FakeExtensionRunner {
  constructor(params) {
    this.params = params;
  }
}
```
That way you could call it more naturally from `createFakeExtensionRunner()`:

```js
const runner = new FakeExtensionRunner(params);
```